docs/reference-architectures/app-service-web-app/baseline-zone-redundant-content.md

```diff
@@ -70,13 +70,13 @@ Application Gateway is a regional resource that meets the requirements of this b
 
 ### Flow from App Services to Azure services
 
-This architecture uses [virtual network integration](/azure/app-service/overview-vnet-integration) for the App Service, specifically to route traffic to private endpoints through the virtual network. The baseline architecture doesn't enable *all traffic routing* to force all outbound traffic through the virtual network, just internal traffic such as traffic bound for private endpoints.
+This architecture uses [virtual network integration](/azure/app-service/overview-vnet-integration) for the App Service, specifically to route traffic to private endpoints through the virtual network. The baseline architecture doesn't enable *all traffic routing* to force all outbound traffic through the virtual network, just internal traffic, such as traffic bound for private endpoints.
```

Comment:
I don't think there is an issue with the comma.

Reply:
Updated.